bundler/lib/bundler/man/bundle-config.1.ronn

```diff
@@ -176,6 +176,9 @@ learn more about their operation in [bundle install(1)](bundle-install.1.html).
    relative paths in the `Gemfile`, among other things. By default, bundler
    will search up from the current working directory until it finds a
    `Gemfile`.
+* `gemfile_lock` (`BUNDLE_LOCKFILE`):
```

Comment:
We have already updated the method in the Gemfile to be `lockfile`, and I think we should do that here so that we have a constant name used across this PR.